integr8ly/integreatly-operator
# MGAPI-244 - Fix failing tests for managed api install

# Description

* Fix AWS alerts tests for managed api install
* Fix Blobstoage test for managed api install
* Update IDP from test and script to allow them be used interchangably

Jira:
* https://issues.redhat.com/browse/MGDAPI-244

## Verification
* Check out this branch
* Provision a **BYOC** cluster
* Install managed-api with **external storage**
```
export INSTALLATION_TYPE=managed-api
export USE_CLUSTER_STORAGE=false
make code/run
```
* Run functional tests from this ticket
```
go clean -testcache && go test -v ./test/functional -run="//(^A16)|(^B03)|(^B04)|(^B06)|(^C04)|(^H03)|(^H07)|(^H11)|(^Verify Network Policy)|(^F04)" -timeout=80m
```
* Verify tests passed
* Run idp script **after** tests
```
PASSWORD=Password1 DEDICATED_ADMIN_PASSWORD=Password1 ./scripts/setup-sso-idp.sh
```
* Try sign into cluster with any created user from script
* Verify there was no auth error when trying to sign in and user signed into the console successfully
## Type of change




## Checklist
